package.xml

```diff
@@ -4,24 +4,25 @@
   <description>
   The car demo
   </description>
-  <maintainer email="tfoote@openrobotics.org">Tully Foote</maintainer>
+  <maintainer email="free.geronimo@hotmail.de">Andreas Baulig</maintainer>
+  <maintainer email="64274389+Braafisch@users.noreply.github.com">Wolfgang Bradfisch</maintainer>
+  <maintainer email="83015830+sunny-5555@users.noreply.github.com">Sungeeta Singh</maintainer>
```

Comment:
@sunny-5555 @Braafisch

Wollt ihr eure E-Mail hier stehen haben?

Reply:
Ja ist in Ordnung!